cloud_experts_tutorials/cloud-experts-using-aws-ack.adoc

```diff
@@ -18,22 +18,24 @@ toc::[]
 //  - Connor Wooley
 //---
 
-link:https://aws-controllers-k8s.github.io/community/[AWS Controllers for Kubernetes] (ACK) lets you define and use AWS service resources directly from {product-title} (ROSA). With ACK, you can take advantage of AWS-managed services for your applications without needing to define resources outside of the cluster or run services that provide supporting capabilities like databases or message queues within the cluster.
+link:https://aws-controllers-k8s.github.io/community/[AWS Controllers for Kubernetes] (ACK) lets you define and use AWS service resources directly from {product-title} (ROSA). With ACK, you can take advantage of AWS-managed services for your applications without needing to define resources outside of the cluster or run services that provide supporting capabilities such as databases or message queues within the cluster.
 
-Users can install various ACK Operators directly from OperatorHub. This makes it relatively easy to get started and start using it with your applications. This controller is a component of the AWS Controller for Kubernetes project. This project is currently in developer preview.
+You can install various ACK Operators directly from OperatorHub. This makes it easy to get started and use the Operators with your applications. This controller is a component of the AWS Controller for Kubernetes project, which is currently in developer preview.
 
-This tutorial shows you how to use the ACK S3 Operator as an example, but can be adapted for any other ACK Operator in the OperatorHub of your cluster.
+Use this tutorial to deploy the ACK S3 Operator. You can also adapt it for any other ACK Operator in the OperatorHub of your cluster.
 
+[id="cloud-experts-using-aws-ack-prerequisites"]
 == Prerequisites
 
 * A ROSA cluster
 * A user account with `cluster-admin` privileges
-* You have access to the OpenShift CLI (`oc`)
-* You have access to the AWS CLI (`aws`)
+* The OpenShift CLI (`oc`)
+* The Amazon Web Services (AWS) CLI (`aws`)
 
```

Comment:
No rosa CLI prereq? I guess the oc CLI will suffice, just wondering why we wouldn't push/suggest the ROSA CLI given it is a ROSA based tutorial? FWIW, I have seen a lot of oc CLI (vs rosa CLI) prereqs in these tutorials so I guess that must be the standard prereq.

Reply:
Yeah it's a valid question. I guess it's cos this tutorial has commands that are either in the `AWS` or `oc` CLIs.